actionpack/test/dispatch/host_authorization_test.rb

```diff
@@ -284,7 +284,7 @@ class HostAuthorizationTest < ActionDispatch::IntegrationTest
     }
 
     assert_response :forbidden
-    assert_match "Blocked host: 127.0.0.1", response.body
+    assert_match "Blocked hosts: www.example.com", response.body
```

Comment:
This and the other test change demonstrate the issue: the displayed blocked host previously would be HTTP_X_FORWARDED_HOST even though that value was already added to `config.hosts`. The correct value here should be HTTP_HOST